src/WebAppDIRAC/WebApp/handler/UPHandler.py

```diff
@@ -301,15 +265,8 @@ def web_listPublicStates(self):
   @asyncGen
   def web_publishAppState(self):
     up = self.__getUP()
-    try:
-      name = self.request.arguments['name'][-1]
-    except KeyError as excp:
-      raise WErr(400, "Missing %s" % excp)
-    try:
-      access = self.request.arguments['access'][-1].upper()
-    except KeyError as excp:
-      access = 'ALL'
-
+    name = self.get_argument("name")
```

Comment:
Why do not raise `WErr` here when argument is absent?

Reply:
`get_argument` raises a `MissingArgumentError` which is a subclass of `HTTPError` so I think there is no need to explicitly raise a `WErr`?

* https://www.tornadoweb.org/en/branch5.1/web.html#tornado.web.RequestHandler.get_argument